deployment-bundle-steps.md

```diff
@@ -0,0 +1,39 @@
+# How to setup a project with deployment to S3
+
+This document assumes you have already setup the S3 bucket.
+
+## Using [CircleCi](https://circleci.com/)
+
+1. Log into [circleci.com](https://circleci.com/) and add your project. Set up the following credentials: (see other members of the team for these values).
+  - Environment Variables
+    - GHUSER
+    - GHPASS
```

Comment:
Until we get something in place to store these kind of credentials, such as vault etc, this is the best choice, for me, to store it. Obviously, we need to keep these creds out of VCS, but happy for other suggestions.